internal/worker/sshserver/server.go

```diff
@@ -206,30 +221,6 @@ func (s *ServerWorker) directTCPIPHandler(srv *ssh.Server, conn *gossh.ServerCon
 	// Since we only need the raw data to redirect, we can discard them.
 	go gossh.DiscardRequests(reqs)
 
-	jumpServerPipe, terminatingServerPipe := net.Pipe()
-
-	s.tomb.Go(func() error {
-		defer ch.Close()
-		defer jumpServerPipe.Close()
-		defer terminatingServerPipe.Close()
-		_, err := io.Copy(ch, jumpServerPipe)
-		if err != nil {
-			s.config.Logger.Errorf("failed to copy data from jump server to client: %v", err)
-		}
-
-		return nil
-	})
-	s.tomb.Go(func() error {
-		defer ch.Close()
-		defer jumpServerPipe.Close()
-		defer terminatingServerPipe.Close()
-		_, err := io.Copy(jumpServerPipe, ch)
-		if err != nil {
-			s.config.Logger.Errorf("failed to copy data from client to jump server: %v", err)
-		}
-
-		return nil
-	})
```

Comment:
Nice to see this go. It also makes it easier to reason about the number of goroutines being managed by the worker.